### PR TITLE
Clarify installation instructions

### PR DESCRIPTION
**Pull request by docsbydesign**
*Issue #, if available:* (N/A)

*Description of changes:*

1. Clarify the installation instructions. The current language seems suggest that the aws-crt common runtime is not required.
2. Add required libraries to build from source.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,24 @@ to JS by the [awscrt](https://github.com/awslabs/aws-crt-nodejs) package.
...
 
-### Common Run Time
-The aws-crt package can be installed via npm 
+### Install the required libraries
+```
+sudo apt-get install cmake
+sudo apt-get install libssl-dev
+```
+
+### Install the common run time
```
Comment (justinboswell) [inline README.md:38]: This step shouldn't be necessary, the CRT should come in as a transitive dependency.

Comment (docsbydesign) [reply README.md:38]: I would have thought so, but on some of my [many] Raspberry Pi installation attempts, I got errors that it couldn't be found. Adding these to the process made the errors go away. It seemed like something that would be [almost] a no-op if the libraries were present and would save the installation if they weren't. Note, that I didn't see any problems when installing the SDK on an Ubuntu EC2 instance. I only saw them on the Raspberry Pi with the May 2020 OS.

Comment (justinboswell) [inline README.md:38]: ```Install the AWS Common Runtime```

**Review by justinboswell — APPROVED** [review]
Fix-n-ship

**Pull request merged**